test/example_data.dart

```diff
@@ -660,7 +674,7 @@ UpdateMessageEvent updateMessageEditEvent(
     messageId: messageId,
     messageIds: [messageId],
     flags: flags ?? origMessage.flags,
-    editTimestamp: editTimestamp ?? 1234567890, // TODO generate timestamp
+    editTimestamp: editTimestamp ?? utcTimestamp(),
```

Comment:
I guess I commented on this change earlier today over on #1498 where Chris had cherry-picked it:
https://github.com/zulip/zulip-flutter/pull/1498#discussion_r2085155042